src/components/MemberArea/MemberArea.js

```diff
@@ -6,15 +6,17 @@ import EditProfile from './EditProfile';
 import PendingApplications from './PendingApplications';
 import LoginNavigation from '../LoginNavigation/LoginNavigation';
 import useWindowSize from '../../utils/useWindowSize';
-import { isMentor, isAdmin } from '../../helpers/user';
+import { isMentor, isAdmin, isMentorAvailable} from '../../helpers/user';
 import UserContext from '../../context/userContext/UserContext';
+import { updateMentorAvailability } from '../../../src/api/index';
+import Switch from '../../components/Switch/Switch';
 
 function MemberArea({ onOpenModal }) {
   const authenticated = auth.isAuthenticated();
+  const isDesktop = useWindowSize().width > 800;
   const [isAuthenticated, setIsAuthenticated] = useState(authenticated);
   const [isMemberMenuOpen, setIsMemberMenuOpen] = useState(false);
-  const isDesktop = useWindowSize().width > 800;
-
+  const [isAvailable, setIsAvailable] = useState((()=> isMentorAvailable()));
```

Comment:
You can use `currentUser` and `updateUser` from [`UserContext`](https://github.com/Coding-Coach/find-a-mentor/blob/master/src/context/userContext/UserContext.js#L9)